docs/pages/preview/upcoming-releases.mdx

```diff
@@ -15,13 +15,10 @@ The Teleport team delivers a new major release roughly every 3 months.
 
 | Version          | Date                | Description                      |
 |------------------|---------------------|----------------------------------|
-| Release          | January 27th, 2023  | Good to go for production.       |
+| Release          | January 31st, 2023  | Good to go for production.       |
```

Comment:
Technically it was published today (the 30th)

Reply:
I just removed it all because this release is out anyway.